recipes/xkbcommon/all/conanfile.py

```diff
@@ -152,8 +144,7 @@ def package_info(self):
             if not self._has_build_profile:
                 self.cpp_info.components["xkbcli-interactive-wayland"].requires.append("wayland-protocols::wayland-protocols")
 
-        if Version(self.version) >= "1.0.0":
-            self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
+        self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
```

Comment:
I don't think this is even necessary anymore. The `bin` directory should get added automatically to `PATH`.

Reply:
It's necessary for legacy generators.